plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergPageSinkProvider.java

```diff
@@ -109,7 +110,7 @@ private ConnectorPageSink createPageSink(ConnectorSession session, IcebergWritab
                 session,
                 tableHandle.fileFormat(),
                 tableHandle.storageProperties(),
-                maxOpenPartitions,
```

Comment:
Is `maxOpenPartitions` used anywhere else? Seems like we should use the session property for those usage sites as well and then remove this field from the class.